OpenRA.Game/Map/MapPreview.cs

```diff
@@ -323,7 +323,8 @@ public void Install()
 						Game.RunAfterTick(() =>
 						{
 							var package = modData.ModFiles.OpenPackage(mapFilename, mapInstallPackage);
-							UpdateFromMap(package, mapInstallPackage, MapClassification.User, null, GridType);
+							if (package != null)
+								UpdateFromMap(package, mapInstallPackage, MapClassification.User, null, GridType);
```

Comment:
Under what circumstances might the package possibly be null after we downloaded the map? If it is null, should we not report some sort of error?